test/functional/device-spec.js

```diff
@@ -1,16 +1,30 @@
 /* eslint no-undef: "off" */
-const assert = require('assert').strict
+const Assert = require('assert').strict
+const dotenv = require('dotenv')
 const StApi = require('../../lib/api')
+const Client = require('../../lib/platform/client')
+
+dotenv.config()
```

Comment:
You can actually change this into a one-liner at the top – e.g.,`require('dotenv').config()`